apiary.apib

```diff
@@ -3489,6 +3492,7 @@ Create a new milestone.
     + Attributes (object)
         + project_id: `1c159f98-4b07-438a-9f42-fb4206b9530d` (string, required)
         + due_on: `2018-01-01` (string, required)
+        + starts_on: `2020-06-15` (string, optional)
```

Comment:
PP: the example date should be tied to the `due_on` date, to give a meaningful example: you cannot have a milestone that starts in 2020 but is due in 2018.

Reply:
Done! :heavy_check_mark: